hazelcast-client/src/main/java/com/hazelcast/client/spi/ClientProxy.java

```diff
@@ -84,6 +85,18 @@ protected final ClientProxy setContext(ClientContext context) {
         return this;
     }
 
+    protected SerializationService getSerializationService() {
+        return serializationService != null ? serializationService : lazySerializationService;
```

Comment:
Normally I do the same, but in this case I tried both variants and in the end it looked smoother to read: `return A != null ? A : B` instead of `return A == null ? B : A`